src/components/cards/TradeCardGP/TradeCardGP.vue

```diff
@@ -246,14 +246,14 @@ export default defineComponent({
 
       if (trading.isGnosisTrade.value) {
         if (trading.gnosis.validationError.value != null) {
-          const errorCode = trading.gnosis.validationError.value;
+          const validationError = trading.gnosis.validationError.value;
```

Comment:
This isn't really an "errorCode" any more - could be a translated string / errorCode.